### ISSUE TITLE
Can't add debugpy config to an existing workspace launch.json file 

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode-python-debugger#66]
Testing #64

1. Be in a workspace which already has a launch.json file (it can even be empty)
2. Click the **Add Configuration...** button 
3. :bug: Debugpy is not an option 
<img width="575" alt="image" src="https://github.com/microsoft/vscode-python-debugger/assets/30305945/d39e8391-e58e-474d-9a3f-d5a4429dc37e">

**Comment by paulacamargo25** [comment]
Thanks for testing Joyce, It seems that the extension has not been activated, could you try to reload vscode, and if it does not work, could you see in the output if it shows any errors?

**Comment by joyceerhl** [comment]
The debugpy extension is indeed not activated. There are no errors. Maybe the Add Configuration button should emit `onDebugInitialConfiguration`?

**Comment by paulacamargo25** [comment]
Makes sense to me, let's see what others think. I'm going to transfer the issue to vscode.
